### DISCUSSION TITLE
Practice Discussion

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#13] [Show and tell]
Use this to practice threaded discussions.

**Comment by u9000-Nine** [comment]
Hello

**Reply by deljennie-a11y** [reply]
Hello!

**Reply by rachaelbradley** [reply]
This is a reply

**Reply by cwadamsoforacle** [reply]
Looks like I am unable to reply to a reply, so limits on how far the thread will go.

**Reply by mgifford** [reply]
Thanks @rachaelbradley

**Reply by ljoakley** [reply]
reply test

**Reply by a11ydoer** [reply]
Hi everyone, This is Jemma!

**Reply by colabottles** [reply]
Replying to the initial comment.

**Reply by rachaelbradley** [reply]
Thank you for trying this out

**Comment by u9000-Nine** [comment]
> Hello!

Howdy

**Reply by u9000-Nine** [reply]
Oh so you can't reply to replies.

**Reply by lauracarlson** [reply]
replying

**Reply by bbrown0** [reply]
reply

**Reply by OliKei** [reply]
again a reply.

**Reply by dav-idc** [reply]
Wow, another reply!

**Comment by eric-hind** [comment]
Replying

**Reply by mgifford** [reply]
So what what happens with @eric-hind's highlight voted on comment?

**Reply by mgifford** [reply]
See the top comment - https://github.com/w3c/wcag3/discussions/13?sort=top

**Comment by ljoakley** [comment]
up arrow?

**Reply by colabottles** [reply]
> up arrow?

To upvote the comment.
